app/lib/pre_assembly/batch.rb

```diff
@@ -43,11 +43,12 @@ def load_skippables
     end
 
     # Runs the pre-assembly process
-    # @return [void]
+    # @return [boolean] true if all objects succeeded, false if any errors
     def run_pre_assembly
       log "\nstarting run_pre_assembly(#{run_log_msg})"
-      process_digital_objects
+      result = process_digital_objects
       log "\nfinishing run_pre_assembly(#{run_log_msg})"
+      result
     end
```

Comment:
if there are any errors that occur, we will return false so we can set the state of the job run as needed